src/Sylius/Bundle/CoreBundle/Resources/config/services.xml

```diff
@@ -237,22 +240,63 @@
             <argument type="service" id="sylius.repository.payment_method" />
         </service>
 
-        <service id="sylius.taxation.order_shipment_taxes_by_zone_applicator" class="%sylius.taxation.order_shipment_taxes_by_zone_applicator.class%">
+        <service id="sylius.taxation.order_shipment_taxes_applicator" class="%sylius.taxation.order_shipment_taxes_applicator.class%">
```

Comment:
Our change of naming here evolved from trying to work through the pattern, I don't think it was an immediate decision, but an attempt at trying to simplify the naming when understanding all of the components involved.

As it stands, I would say KISS: If someone wants an alternative mechanism that falls outside of Sylius's now wide  scope of functionality then let them implement and name it accordingly. Even for the simplest of situations most people would just create a single tax zone and use this rather than implement their own version.

If you disagree then of course I'm not going to lose any sleep if you decide to change it back ;)